graph/utils.go

```diff
@@ -87,6 +87,7 @@ func ConvertHttpReq(r models.HttpReq) *model.HTTPReq {
 		Header:     header,
 		Method:     ConvertMethod(r.Method),
 		Body:       r.Body,
+		URL:        &r.URL,
```

Comment:
this seems to be redundant from your previous PR.

Reply:
Done